environs/bootstrap/bootstrap.go

```diff
@@ -479,15 +511,10 @@ func bootstrapImageMetadata(
 	}
 
 	if bootstrapImageId != "" {
-		arches := availableTools.Arches()
-		if len(arches) != 1 {
-			return nil, errors.NotValidf("multiple architectures with bootstrap image")
+		if bootstrapSeries == nil {
+			return nil, errors.NotValidf("no series specified with bootstrap image")
 		}
-		allSeries := availableTools.AllSeries()
-		if len(allSeries) != 1 {
-			return nil, errors.NotValidf("multiple series with bootstrap image")
-		}
-		seriesVersion, err := series.SeriesVersion(allSeries[0])
+		seriesVersion, err := series.SeriesVersion(*bootstrapSeries)
```

Comment:
is bootstrap series always not nil? it is optional on the CLI
what's the rational for changing the logic here?